FormSubmissionAndValidation2/src/App.jsx

```diff
@@ -0,0 +1,10 @@
+import './App.css'
+import { UserForm } from './components/UserForm'
```

Comment:
Why not use a default import/export here?

Reply:
could have used default as well. 
My understanding is that from a file there can be only one default export but multiple other exports.

Reply:
Yes, only one default export, which should be the main component defined in that file. Most of your files don't have a default export at all.